### PR TITLE
Update version due to libunwind update

### PR DESCRIPTION
**Pull request by AdrianCX**
Updated both fortanix-sgx-tools and aesm-client versions

### CODE REVIEW

**Comment by jethrogb** [comment]
Can you update the aesm-client version as well?

File: doc/FORTANIX-SGX-ABI.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 This document describes the ABI of SGX enclaves built using `libenclave`.
 
+## Enclave runner compatibility
```
Comment (jethrogb) [inline doc/FORTANIX-SGX-ABI.md:5]: This is not related to the runner. It's the elf2sgxs conversion

File: doc/FORTANIX-SGX-ABI.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+
+a. rust-lang toolchain version: https://github.com/rust-lang/rust/blob/master/src/libstd/sys/sgx/abi/entry.S
+```
+2:  .long 0                    /* desc - toolchain version number, 32-bit LE */
+```
```
Comment (jethrogb) [inline doc/FORTANIX-SGX-ABI.md:10]: Instead of pointing at a source file, can you describe this in terms of ELF sections?

File: doc/FORTANIX-SGX-ABI.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
...
+
+b. fortanix-sgx-tools version: https://github.com/fortanix/rust-sgx/blob/master/fortanix-sgx-tools/Cargo.toml
```
Comment (jethrogb) [inline doc/FORTANIX-SGX-ABI.md:12]: Don't need this

File: doc/FORTANIX-SGX-ABI.md [hunk 4]
```diff
@@ -2,6 +2,21 @@
...
+
+| rust-lang toolchain version | fortanix-sgx-tools version | 
+| ---------------------------:| --------------------------:|
+|             1               |          0.4.0             |
+|             0               |          0.3.1             |
```
Comment (jethrogb) [inline doc/FORTANIX-SGX-ABI.md:17]: Can you please add a reference to the Rust std version (by git commit) as well (see below for ABI compat)

Comment (AdrianCX) [reply doc/FORTANIX-SGX-ABI.md:17]: that is not delivered yet, rust-lang comes after rust-sgx, once it is merged I will add commit as well..

Comment (jethrogb) [reply doc/FORTANIX-SGX-ABI.md:17]: Ok but you can add the column and fill it in for version 0. You can say TBD for version 1

File: doc/FORTANIX-SGX-ABI.md (hunk 1 above)
Comment (AdrianCX) [reply doc/FORTANIX-SGX-ABI.md]: updated, thanks

File: doc/FORTANIX-SGX-ABI.md (hunk 2 above)
Comment (AdrianCX) [reply doc/FORTANIX-SGX-ABI.md]: done

File: doc/FORTANIX-SGX-ABI.md (hunk 3 above)
Comment (AdrianCX) [reply doc/FORTANIX-SGX-ABI.md]: thanks

**Comment by jethrogb** [comment]
I think I used to think to keep the minor version in sync between enclave-runner, fortanix-sgx-tools and fortanix-sgx-abi. I'm not sure though. Let me think about it.

File: aesm-client/Cargo.toml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aesm-client"
-version = "0.4.0"
+version = "0.4.1"
```
Comment (jethrogb) [inline aesm-client/Cargo.toml:3]: Please undo this change, will conflict with #240

File: doc/FORTANIX-SGX-ELF.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+- `HEAP_BASE`. Size 8 bytes. The base address (relative to enclave start) of
+  the heap area, little-endian.
+- `HEAP_SIZE`. Size 8 bytes. The heap size in bytes, little-endian.
+- `ENCLAVE_SIZE`. Size 8 bytes. The enclave size in bytes, little-endian.
+- `CFGDATA_BASE`. Size 8 bytes. The base address (relative to enclave start) of
+  the enclave configuration area, little-endian.
+- `RELA`. Size 8 bytes. Value of the RELA entry in the dynamic table.
+- `RELACOUNT`. Size 8 bytes. Value of the RELACOUNT entry in the dynamic table.
+- `DEBUG`. Size 1 byte. Non-zero if debugging is enabled, zero otherwise.
```
Comment (jethrogb) [inline doc/FORTANIX-SGX-ELF.md:61]: Please describe all missing variables

Comment (AdrianCX) [reply doc/FORTANIX-SGX-ELF.md:61]: Fixed, thanks

File: aesm-client/Cargo.toml (hunk 5 above)
Comment (AdrianCX) [reply aesm-client/Cargo.toml]: reverted file
